src/Aquifer.Public.API/Endpoints/ParentResources/List/Request.cs

```diff
@@ -0,0 +1,14 @@
+namespace Aquifer.Public.API.Endpoints.ParentResources.List;
+
+public sealed class Request
```

Comment:
Do we also want to allow filtering by `ResourceType`?

Reply:
If you mean by "Dictionary" or "Guide", I'd say might as well.

Reply:
Ok, I'll add that in.